### PR TITLE
JBPM-9273 System error occurred while adding deployment unit

### PR DESCRIPTION
**Pull request by bxf12315**
**Thank you for submitting this pull request**

**JIRA**: [JBPM-9273](https://issues.redhat.com/browse/JBPM-9273)

**Referenced Pull Requests**:

**Business Central**: [WAR file](https://donwnload.here/something)

**VS Code**: [plugin](https://donwnload.here/something)

<pre>
To retest a PR or trigger a specific build please add a comment:

* Retest PR: <b>jenkins retest this</b>
* A full downstream build: <b>jenkins do fdb</b>
* A compile downstream build: <b>jenkins do cdb</b>
* A full production downstream build: <b>jenkins do product fdb</b>
* An upstream build: <b>jenkins do upstream</b>
</pre>

### CODE REVIEW

**Comment by bxf12315** [comment]
jenkins do fdb

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/resources/org/kie/workbench/common/screens/server/management/client/resources/i18n/Constants.properties [hunk 1]
```diff
@@ -151,3 +151,4 @@ NewContainer.GAVNotExist=GAV '{0}' not found in the Maven repository.
 NewContainer.GAVNotExist.Save=GAV '{0}' not found in the Maven repository. Are you sure you want to deploy it? 
 NewContainer.Save=Ok
 NewContainer.SaveContainerSpec=Save Container Spec
+NewContainer.FailedContainer=The Container failed on kie server, please confirm it
```
Comment (nmirasch) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/resources/org/kie/workbench/common/screens/server/management/client/resources/i18n/Constants.properties:154]: @bxf12315 I don't get this "please confirm it", is there something the admin have do here?  on this message is just to let the admin know that the container is failing? I guess just notifying the failing would be enough. Wdyt?

Comment (bxf12315) [reply kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/resources/org/kie/workbench/common/screens/server/management/client/resources/i18n/Constants.properties:154]: Done

**Comment by bxf12315** [comment]
@nmirasch @prashuts updated, please review it.

**Comment by prashuts** [comment]
jenkins do fdb

**Comment by bxf12315** [comment]
jenkins do fdb

**Comment by prashuts** [comment]
@bxf12315 i followed the steps as in new video and i could not see error message.Please check and also check on files in conflicting PR

**Comment by bxf12315** [comment]
jenkins do fdb

**Comment by bxf12315** [comment]
jenkins do fdb

**Pull request closed without merging**